test/tests.cpp

```diff
@@ -5465,7 +5465,6 @@ TEST_CASE_METHOD(QInterfaceTestFixture, "test_mirror_circuit_19", "[mirror]")
     qftReg->H(0);
     qftReg->H(2);
     qftReg->IT(2);
-    qftReg->CNOT(0, 2);
```

Comment:
This was an asymmetry intentionally left behind in `test_mirror_circuit_19`, necessary as all other lines to achieve the bug, to the best of my ability to simplify the procedurally generated test. This requires that qubit `0` is identically |0>. I have no clue if it should be, honestly.

Reply:
It raises my attention because, hypothetically, the new co-opted state preparation `AI()` methods could trip some kind of "Gimbal lock," for the separation we're doing. But, it's speculative to do or say anything more than check whether this particular line expects identically |0> for a control bit.

Reply:
Perfectly enough, it's empirically not quite exactly `1.0` amplitude for |1>, and the target bit is |+>. _[screaming heard in the distance]_

Reply:
It think it's exactly `1.0`, (with an unlucky first roll). It seems that the cache is out of sync, and a public bit probability check would fix it, but that isn't the acceptable protected API method solution.